_pages/iframe.html

```diff
@@ -0,0 +1,33 @@
+---
+permalink: /iframe/
+layout: null
+sitemap: false
+published: false
+---
+<!doctype html>
+<html class="no-js" lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style>
+      iframe {
+        width: 100%;
+        height: 100vh;
+      }
+    </style>
+  </head>
+  <body>
+    <p>This is a testing harness. Browse to
+      <code>
+        <a href="/iframe/?page=/conduct/">/iframe/?page=/conduct/</a>
+      </code>
+      to test the conduct page in an iframe.
+    </p>
+   <iframe id="iframe-test" src=""></iframe>
+  </body>
+  <script>
+    const iframe = document.getElementById('iframe-test')
+    iframe.src = 'http://localhost:4000'+(new URLSearchParams(window.location.search)).get('page')
```

Comment:
This value will have to change before we merge.

Reply:
This is meant for local testing, not production.

Reply:
@jefftriplett Any concerns with me merging this as an unpublished page meant for local testing? I can remove it as well. I'm not aware of us having a staging site for us to test changes like this before actually releasing them.

Reply:
@tim-schilling nope, test away.